sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/ExtensionMethodShouldBeInSeparateNamespace.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace SomeNamespace
```

Comment:
Let's add tests for global namespace. Mainly for documentation purposes, as I think the code should work fine.

Reply:
Good catch.

Reply:
Another corner case: extension methods in nested classes.